src/jit/hwintrinsicxarch.cpp

```diff
@@ -139,7 +145,15 @@ NamedIntrinsic Compiler::lookupHWIntrinsic(const char* methodName, InstructionSe
                 result = hwIntrinsicInfoArray[i].intrinsicID;
             }
         }
+
+        if ((result > NI_SW_INTRINSIC_START) && (result < NI_HW_INTRINSIC_END))
```

Comment:
Shouldn't this be `result < NI_SW_INTRINSIC_END`

Reply:
Yes, it sounds better, but then I would add NI_HW_INTRINSIC_END with value equal to NI_SW_INTRINSIC_START

Reply:
I think they should be uniquely defined values to help prevent confusion